CODEYARDS/DISCORD-LINKS/links

```diff
@@ -0,0 +1,2 @@
+This a great choice for a decentralized web curriculum effort! - https://www.pubpub.org/ 
```

Comment:
great idea to store this, can we have a working group for Curriculum tho instead of putting it in codeyards?